### DISCUSSION TITLE
[Question]It's too difficult! What do you have to go through to successfully execute the `Flet build apk` command and package an Android application in mainland China? Just take a look at the screenshot below!

### DISCUSSION THREAD

**Discussion by LegendaryPistachio** [flet-dev/flet#4365] [Q&A]
### Question

We strongly recommend the Flet framework official team to enhance the localization support for executing the `Flet build apk` command successfully for packaging Android applications in mainland China in the near future. For example, the first solution: provide a list of environment dependency version requirements or an official tutorial for executing `Flet build apk` command successfully with Flet 0.24.1. Ideally, the official team could launch a localized Flet build apk packaging integration tool that includes the basic dependency configurations without needing internet access. Network access would only be required when downloading the integrated packaging tool or during version upgrades. There should be no need for internet during the Android app packaging process. The second solution: the official team could establish a cloud packaging environment where users can upload their projects and download the packaged apk files without any download limits. The third solution: the official team could adopt a multi-path strategy that includes command packaging, localized packaging, and cloud packaging.

![20241114090623](https://github.com/user-attachments/assets/48b05207-695a-4be5-a2c9-0d170bbe156c)

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by LegendaryPistachio** [comment]
Request Feodor Fitsner to respond.

**Comment by tanmay-bhatgare** [comment]
In the pre version of flet, they made debugging and packing apk easier with new packaging method. You can check that on [official blog](https://flet.dev/blog/flet-new-packaging-pre-release/), upgrading to pre-flet might be helpful for packaging app.
